src/Trains.NET.Rendering/Terrain/TerrainColourLookup.cs

```diff
@@ -6,8 +6,9 @@ namespace Trains.NET.Rendering
     public static class TerrainColourLookup
     {  
         public static Color DefaultColour => Colors.Green;
+        private static Color WaterToLandTransition = Colors.LightYellow;
 
-        private static readonly List<Color> s_heightOrderedColours = new List<Color> {
+        private static readonly List<Color> _heightOrderedColours = new List<Color> {
```

Comment:
Why did you rename `s_heightOrderedColours` to `_heightOrderedColours`? now I'm just gonna get squiggles and have to rename it back. Did you not get squiggles after renaming it locally?